prdoc/schema_user.json

```diff
@@ -0,0 +1,228 @@
+        "type": "object",
+        "properties": {
+          "db": {
+            "type": "array",
+            "nullable": false,
+            "description": "List of the Database Migrations or []",
+            "items": {
+              "$ref": "#/$defs/migration_db"
+            },
+            "minItems": 0,
+            "required": [
+              "name",
+              "description"
+            ]
+          },
+          "runtime": {
+            "type": "array",
+            "nullable": false,
+            "description": "List of the Runtime Migrations or []",
+            "minItems": 0,
+            "items": {
+              "$ref": "#/$defs/migration_runtime"
+            },
+            "required": [
+              "db",
+              "runtime"
+            ]
+          }
+
+        }
+      },
+      "host_functions": {
+        "description": "Lits of the host functions involved in this PR.\nThis is used to generate the host functions documentation.",
+        "type": "array",
+        "minItems": 0,
+        "items": {
+          "$ref": "#/$defs/host_function"
+        }
+      }
+    },
+    "required": [
+      "title",
+      "doc",
+      "crates",
+      "migrations",
+      "host_functions"
+    ],
+    "$defs": {
+      "audience": {
+        "oneOf": [
+          {"const": "Node Dev", "title": "Someone working on the node"},
+          {"const": "Runtime Dev", "title": "Someone working on runtimes, frames, pallets"},
+
+          {"const": "Node Operator", "title": "Someone running a node in general, not specific to validators"},
+          {"const": "Validator", "title": "Someone running a Validator node"},
+
+          {"const": "Parachain Builder", "title": "Someone building a parachain"},
+
+          {"const": "App Builder", "title": "Someone building Apps and dApps on the shoulder of the Polkadot network and its parachains"},
+
+          {"const": "Runtime User", "title": "Someone involved or interested in the governance process or noticeable changes to the runtime and/or its dependencies"}
+
+        ]
+      },
+      "crate": {
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "semver": {
+            "$ref": "#/$defs/semver"
+          },
+          "note": {
+            "type": "string"
+          }
+        }
+      },
+      "semver": {
+        "type": "string",
+        "enum": [
+          "major",
+          "minor",
+          "patch",
+          "no change"
+        ]
+      },
+      "migration_db": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "description": {
+            "type": "string"
+          }
+        },
+        "additionalProperties": false,
+        "required": [
+          "name",
+          "description"
+        ]
+      },
```

Comment:
These should be optional parts of the individual audiences.

Reply:
What will you do with this information that there is a db migration?

A proper prdoc would mention this migration in the proper audience any way to tell the user that it is there. If you only want this for some kind of filtering, fine. However then just add it as optional fields to audience and just a bool to indicate that there is either a db migration or a runtime migration.

Reply:
The prdoc just holds the information, it does not know how we use it. The prdoc file does notthing.

During the release, all the prdoc files involved in the release will be fed into one or more templates (tbd).
Those will define what we actually do with the informations.

For the  example with `migration_db`, this will be shown to some audiences.
@kianenigma gave some more examples [here](https://forum.parity.io/t/pr-documentation-prdoc/2065/28?u=wilfried_kopp). For the `migration_db`, the information will be shown to the `Runtime dev` only.